phylogenetic/rules/prepare_sequences.smk

```diff
@@ -64,7 +64,8 @@ rule filter:
     params:
         strain_id_field = config["strain_id_field"],
         min_length = config['filter']['min_length'],
-        exclude = config['filter']['exclude']
+        exclude = config['filter']['exclude'],
```

Comment:
_tangent_

Just realized that the `exclude` is included in both the `input` and the `params`. The shell command is using `input.exclude` so we should remove the param to prevent any future confusion.

Reply:
Gah, thanks for flagging. Fixed in https://github.com/nextstrain/lassa/pull/25/commits/2f08f710753a8e965cdb34f038ca286edc4a3019

Reply:
And fixed CI in https://github.com/nextstrain/lassa/pull/25/commits/422e61fa3634a07b86a05e412dd9b7692e60877e